src/components/admin/personen/PersonLock.vue

```diff
@@ -167,6 +167,22 @@
     },
     { immediate: true },
   );
+
+  watch(dialogIsActive, (newDialogIsActive: boolean) => {
+    if (newDialogIsActive) {
+      const manualLock: UserLock | null =
+        props.person.person.userLock?.find(
+          (lock: UserLock) => lock.lock_occasion === PersonLockOccasion.MANUELL_GESPERRT,
+        ) ?? null;
+
+      if (manualLock) {
+        isUnbefristet.value = !manualLock.locked_until;
+        selectedBefristung.value = manualLock.locked_until;
+      }
+    } else {
+      resetBefristungFields();
+    }
+  });
```

Comment:
Nested ifs sind nicht so schön zu sehen imo. Ein Early return ganz am Anfang  und darunter der Rest des Codes sollte auch klappen:

```suggestion
watch(dialogIsActive, (isActive: boolean) => {
  if (!isActive) {
    resetBefristungFields();
    return;
  }

  const manualLock: UserLock | null = props.person.person.userLock?.find(
    (lock: UserLock) => lock.lock_occasion === PersonLockOccasion.MANUELL_GESPERRT
  )  ?? null;;

  if (manualLock) {
    isUnbefristet.value = !manualLock.locked_until;
    selectedBefristung.value = manualLock.locked_until;
  }
});
```